build.bash

```diff
@@ -53,14 +53,16 @@ get_git_repo() {
 ##
 inject_build_repo() {
   if [ -z "${clone_string+x}" ]; then
-    echo_process "inject_build_repo() invoked without clone_string varible set, exiting...."
+    echo_process "inject_build_repo() invoked without clone_string variable set, exiting...."
     exit 1
   fi
   sed -i '$a /usr/bin/apt-get -y install figlet &>/dev/null' "$1"
   sed -i '$a echo "#!/bin/sh\n\ntest -x /usr/bin/figlet || exit 0\n\nfiglet \"Test build, Do not use!\" -w 55" > /etc/update-motd.d/04-test-build-text' "$1"
   sed -i '$a chmod +rx /etc/update-motd.d/04-test-build-text' "$1"
   sed -i '$a echo "$(timestamp) [openHABian] Warning! This is a test build."' "$1"
-  sed -i "s@master https://github.com/openhab/openhabian.git@$clone_string@g" "$1"
+#  sed -i "s@master https://github.com/openhab/openhabian.git@$clone_string@g" "$1"
+  sed -i "s@master@$clonebranch@g" "/etc/openhabian.conf"
```

Comment:
maybe safer to use "s@^clonebranch=.*@clonebranch=$clonebranch@g"

Reply:
ok